pom.xml

```diff
@@ -46,6 +46,17 @@
         </license>
     </licenses>
 
+    <developers>
```

Comment:
Conflicts with `developers` section added separately, to have sufficient metadata for deploying 2.0.0 to sonatype. Very happy for you to update the existing one (including role "founder" obviously).